requirements.txt

```diff
@@ -5,10 +5,10 @@
 
 ## IMPORTANT: Keep aligned with setup.cfg
 
-setuptools
-altgraph
-pefile; sys_platform == 'win32'
-pywin32-ctypes; sys_platform == 'win32'
-macholib; sys_platform == 'darwin'
+setuptools==46.1.3
```

Comment:
Has the config file changed? I don't remember this from previous pyup PRs. I'm not sure what makes sense -- to version these (better for CI) or not (better for development/testing). @htgoebel, @Legorooj, any thoughts?

Reply:
https://pyup.io/docs/bot/config/#pinning

Reply:
I think we should leave them pinned.

Reply:
I'm not sure -- let's leave them unpinned for now, but think about this more.

Reply:
I'll unpin them.